.github/workflows/release.yml

```diff
@@ -3,7 +3,7 @@ name: Release
 on:
   push:
     branches:
-      - main
+      - 1-legacy
```

Comment:
I think we can include both of these? This action would then run on pushes to `main` or `1-legacy`, which would keep us from updating this workflow all the time.